.gitignore

```diff
@@ -1,3 +1,6 @@
 /build
 /docs
 /vendor
+/tests/js/sample
+/tests/css/sample
```

Comment:
These `tests/*/sample` directories must not be ignored: the source files for a lot of the tests are in there.
Only the target paths should be ignored, but those should already have a .gitignore file in those specific folders, where applicable.